README.md

```diff
@@ -1,2 +1,25 @@
-# advisory-group
-Enterprise Advisory Group
+# Enterprise Advisory Group
+
+The Enterprise Advisory Group consists of corporate representatives in leadership roles responsible for managing for large-scale deployments of Node.js in an Enterprise environment. We strive to provide timely feedback to the [TSC](https://github.com/nodejs/TSC), the [CommComm](https://github.com/nodejs/community-committee), the [Node.js project](https://github.com/nodejs/node) and the Node.js Foundation about the needs of Node.js Enterprise users.
+
+Thank you for your interest and support. Please [create an issue](https://github.com/nodejs/user-feedback/issues) in this repo if you are interested in dedicating 5+ hours a month during the formation process.
+
+### Governance and Contributing
+The Enterprise Advisory Group has adopted the core governance and contributing policies of Node.js. This group is a part of the [Node.js End User Feedback Initiative](https://github.com/nodejs/user-feedback) which is under the charter of the [Node.js Community Committee (CommComm)](https://github.com/nodejs/community-committee).
```

Comment:
As this group is planning to be spun up under the End User Feedback initiative it might make sense to have some of the higher level questions regarding how feedback is discussed within that initiative. It could make a bunch of sense to use the EAG to workshop different ideas, but I think having a single consistent methodology would really help

Reply:
Agree that there should be some coordination on this. It's too early to say what those processes will be.